package com.erikjarquin.ventas.model.dto.UserRole;


public class UpdateUserRequest {
    private String name;
    private String email;
    private Long roleId;
    private boolean active;

    //Getter y setter
    public String getName(){
        return name;
    }

    public void setName(String name){
        this.name = name;
    }

    //Getter y setter
    public String getEmail(){
        return email;
    }

    public void setEmail(String email){
        this.email=email;
    }

    //Getter y setter
    public Long getRoleId(){
        return roleId;
    }

    public void setRoleId(Long roleId){
        this.roleId=roleId;
    }

    //Getter y setter
    public boolean isActive(){
        return active;
    }

    public void setActive(boolean active){
        this.active=active;
    }
}
